en/README.md

```diff
@@ -0,0 +1,1138 @@
+  "params": {
+    "fetchId": "1",
+    "lastInfo": "201504011759",
+    "events": [
+      {
+        "extendedInfo": "sampel extended info",
+        "brief": "example brief",
+        "eventId": "1",
+        "time": "20150323151300",
+        "type": "GOOD",
+        "triggerId": 2,
+        "status": "OK",
+        "severity": "INFO",
+        "hostId": 3,
+        "hostName": "exampleName"
+      }
+    ]
+  },
+  "method": "putEvents",
+  "jsonrpc": "2.0"
+}
+```
+
+***Result(result)***
+
+Receive result as result object value whether sent data has been updated. Look at the [[Values](#user-content-putresult)].
+
+```json
+{
+  "id": 1,
+  "result": "SUCCESS",
+  "jsonrpc": "2.0"
+}
+```
+
+### putHostParents(method)
+
+ When HAP complates to connect to Hatohol server,  send all host parent relations to use "ALL" option to Hatohol server.
+
+In case of use "UPDATE" option, send diffarence of host parent relations that based on lastInfo that got from [getLastInfo](#user-content-getlastinfomethod) or saved in internal to Hatohol server. 
+
+***Request(params)***
+
+|Object name|Type |M/O|Default value|Description|
+|:-----------------|:--|:-:|:----------:|:---|
+|hostParents  |object array|M|-|This is object array that stores host parent relations information. Look at the following table for details.|
+|updateType|string    |M|-|Select sending option that match the situation from [[List](#user-content-updatetype)].|
+|lastInfo    |String32767|O|-|Insert to here host parent relation information of last. This information is result of [getLastInfo](#user-content-getlastinfomethod).|
+
+***hostParents object***
+
+ In case of remove host parent relation from Hatohol server, can remove host parent relation that is to sent childHostId by insert empty string to parentHostId.
+
+|Object name|Type |M/O|Default value|Description|
+|:-----------------|:--|:-:|:----------:|:---|
+|childHostId |String255|M|-|Child host id.|
+|parentHostId|String255|M|-|Parent host id.|
+
+```json
+{
+  "id": 1,
+  "params": {
+    "lastInfo": "201504152246",
+    "updateType": "ALL",
+    "hostParents": [
+      {
+        "parentHostId": "10",
+        "childHostId": "12"
+      },
+      {
+        "parentHostId": "20",
+        "childHostId": "11"
+      }
+    ]
+  },
+  "method": "putHostParent",
+  "jsonrpc": "2.0"
+}
+```
+
+***Result(result)***
+
+Receive result as result object value whether sent data has been updated. Look at the [[Values](#user-content-putresult)].
+
+```json
+{
+  "id": 1,
+  "result": "SUCCESS",
+  "jsonrpc": "2.0"
+}
+```
+
+### putArmInfo(method)
+
+Standard motion is to send after each to send host, trigger, event information to Hatohol server, but can send anytime. The minimum interval is 1 second(MUST), The maximum interval is twice of polling interval second that got by the [getMonitoringServerInfo](#user-content-getmonitoringserverinfomethod) and [updateMonitoringServerInfo](#user-content-updatemonitoringserverinfomethod)(SHOULD).
```

Comment:
> but can send anytime. 

but users can send at the arbitrary point of the time. 

Arbitrary is more suitable here, too.